waveorder/stokes.py

```diff
@@ -0,0 +1,344 @@
+"""
+Overview
+--------
+
+This module collects Stokes- and Mueller-related calculations. 
+
+The functions are organized into three groups:
+
+1) A forward function group: 
+
+A = A_matrix(swing, scheme="5-State")
+s0, s1, s3, s3 = s0123_CPL_after_ADR(ret, ori, tra, dop)
+s0, s1, s2 = s012_CPL_after_AR(ret, ori, dop)
+
+2) An inverse function group:
+
+ret, ori, tra, dop = inverse_s0123_CPL_after_ADR(s0, s1, s2, s3)
+ret, ori, tra = inverse_s012_CPL_after_AR(s0, s1, s2)
```

Comment:
I suggest:
ret, ori, tra, dop = inverse_ADR(s0,s1,s2,s3,input='CPL')
ret, ori, tra = inverese_AR(s0,s1,s2,input='CPL')

Reply:
Similar to above...how do these sound:
```
ret, ori, tra, dop = inverse_stokes_after_ADR(s0, s1, s2, s3, input="CPL")
ret, ori, tra = inverse_stokes012_after_AR(s0, s1, s2, input="CPL")
```

`inverse_ADR` alone might make me think that I'm going to get a Mueller matrix.

Reply:
Further down you suggest `estimate_ADR`, and I do like the word `estimate` here. 

I'm now favoring 
```
ret, ori, tra, dop = estimate_ADR_from_stokes(s0, s1, s2, s3, input="CPL")
ret, ori, tra = estimate_AR_from_stokes012(s0, s1, s2, input="CPL")
```